ciphey/basemods/Checkers/human.py

```diff
@@ -1,26 +1,36 @@
 from typing import Dict, Optional
 
 from ciphey.iface import Checker, Config, ParamSpec, registry
+from rich.console import Console
+
+console = Console()
 
 
 @registry.register
 class HumanChecker(Checker[str]):
-    @staticmethod
-    def getParams() -> Optional[Dict[str, ParamSpec]]:
-        pass
 
-    def check(self, text: str) -> Optional[str]:
+    """
+    Uses the person's decision to determine plaintext
+    """
+
+    def check(self, ctext: str) -> Optional[str]:
         with self._config().pause_spinner_handle():
-            response = input(f"Result {text.__repr__()} (y/N): ").lower()
+            response = console.input(
+                f"Possible plaintext: [blue bold]{ctext.__repr__()}[/blue bold] ([green]y[/green]/[red]N[/red]): "
```

Comment:
I should change y/N to yes/NO, and maybe make "possible plaintext" into a question for non-english natives :-)

Reply:
`y/N` is perfectly fine because that's how it's always stated in CLI programs. You either type `y` or `n` and `n` is capitalized because it's the default. I also think that `possible plaintext` is clear enough for people.